doc/ref_arch/kubernetes/chapters/chapter01.rst

```diff
@@ -58,58 +95,97 @@ Individual contributors who are also active in the CNCF TUG (Telecom User Group)
 Exceptions
 ~~~~~~~~~~
 
-Anuket specification define certain policies and `principles <../../../common/chapter00.md#2.0>`__ and strives to coalesce the industry towards conformant Cloud Infrastructure technologies and configurations. With the currently available technology options, incompatibilities, performance and operator constraints (including costs), these policies and principles may not always be achievable and, thus, require an exception process. These policies describe how to handle `non-conforming technologies <../../../common/policies.md#cntt-policies-for-managing-non-conforming-technologies>`__. In general, non-conformance with policies is handled through a set of exceptions (please also see `Exception Types <../../../gov/chapters/chapter09.md#exception-types>`__).
+Anuket specification define certain policies and `principles <../../../common/chapter00.md#2.0>`__ and strives to
```

Comment:
@CsatariGergely I know that the purpose is not to correct grammar but can we include the following minor corrections. Thanks
```suggestion
Anuket specifications define certain policies and `principles <../../../common/chapter00.md#2.0>`__ and strive to
```